### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghost**
Reorganize ECIP Status terms for clarity and to make them easier to follow. (Proposed this change both on ethereumclassic ECIPs and etclabscore ECIPs).

### CODE REVIEW

File: README.md
Comment (stevanlohja) [inline README.md]:
```suggestion
* **Rejected** - Reasons for rejecting ECIPs include duplication of effort, disregard for formatting rules, unfocused or too broad, being technically unsound, not providing proper motivation, or conflicting with the decentralization, backward compatibility, and state finality of the network.
```
"ETC philosophy" doesn't describe any specification unless defined. Might as well list the design principles.

File: README.md
Comment (stevanlohja) [inline README.md]:
```suggestion
* **Withdrawn** - ECIP authors may decide to change the status between Draft, Deferred, or Withdrawn. The ECIP editor may also change the status to Deferred if no progress is being made on the ECIP.
```
More concise suggestions.

File: README.md
Comment (stevanlohja) [inline README.md]: Obsolete ECIP can lead to confusing since it's already related to Replaced. Imagine reintroducing Obsolete ECIPs. To omit and ecip requires an ecip. The content of the ECIP already makes sense of obsoleting anything.

File: README.md
Comment (stevanlohja) [inline README.md]:
```suggestion
 - **For all other ECIPs**, open a PR changing the state of your ECIP to 'Final'. An editor will review your draft and ask if anyone objects to it being finalized. If the editor decides there is no rough consensus - for instance, because contributors point out significant issues with the ECIP - they may close the PR and request that you fix the issues in the draft before trying again.
```

**Review by stevanlohja — CHANGES_REQUESTED** [review]
I've made some suggested commits w/ comments.

File: README.md
Comment (ghost) [reply README.md]: The text is from ECIP-1000 which is meant to be a specification, but i agree it's broad and vague. Might as well leave it out because listing the principles would make it too long.

File: README.md
Comment (ghost) [reply README.md]: Perfect.

File: README.md
Comment (ghost) [reply README.md]: The text is from ECIP-1000, but I agree, let's leave only replaced which implies 'obsolete' as well.

**Review by stevanlohja — APPROVED** [review]

**Comment by stevanlohja** [comment]
LGTM we need more approvals to merge.

File: README.md
Comment (soc1c) [inline README.md]: I don't know why this change is required. Technically, a submitted EIP only becomes a draft after being merged. Before it, it's just _in progress_.

File: README.md
Comment (soc1c) [inline README.md]: That added sentence duplicates logic from _final_.

File: README.md
Comment (soc1c) [inline README.md]: That reads as if a final ECIP can not have any implementation, which is technically incorrect. Final should always mark ECIPs that are beyond accepted, implemented, and in a state where it can no longer be modified.

File: README.md
Comment (soc1c) [inline README.md]: This is also not clean, core proposals can only be final once activated on a network, so after a hardfork on testnet or mainnet.

File: README.md
Comment (meowsbits) [inline README.md]:
```suggestion
 - **For all other ECIPs**, open a PR changing the state of your ECIP to 'Final'. An editor will review your draft and ask if anyone objects to its finalization. If the editor decides there is no rough consensus - for instance, because contributors point out significant issues with the ECIP - they may close the PR and request that you fix the issues in the draft before trying again.
```

File: README.md
Comment (meowsbits) [inline README.md]:
```suggestion
* **Rejected** - Reasons for rejecting ECIPs include duplication of effort, disregard for formatting rules, unfocused or too broad, being technically unsound, not providing proper motivation, or obvious popular disapproval.
```

**Review by meowsbits — DISMISSED** [review]

File: README.md
Comment (ghost) [reply README.md]: Thanks for clarifying. Returning to original text.

Comment (DonaldMcIntyre) [reply README.md]: Thanks for clarifying. Returning to original text.

File: README.md
Comment (ghost) [reply README.md]: It is a line that serves as a transition to the next status. In the current text, several status descriptions seemed the same and not very clear. As long as it is a duplication and serves as clarification, but is not technically wrong, I would leave it.

File: README.md
Comment (ghost) [reply README.md]: For some reason both Ethereum EIP and Ethereum Classic ECIP Status Terms have divided the "Final" status into two: "Final (non-core)" and "Final (Core)". 

It seems there is and intermediate period between "Accepted" and "Final"  where devs implement changes, but are not scheduled or deployed in a hard fork.

If that intermediate period is true or really exists, then we can change the term "Final (non-Core)" to "Intermediate" or "Pre-final" or "Pre-Fork".

If not we can strike "Final (non-Core)" altogether and just leave "Final".

Please let me know if that intermediate state actually exists or what terminology you would use.

File: README.md
Comment (ghost) [reply README.md]: Agreed. Would this sentence work?

"**Final (Core)** - An ECIP that the Core Devs have decided to implement and has been deployed in a hard fork."

File: README.md
Comment (soc1c) [reply README.md]: Yeah, I think we should simplify it here.

Draft > Last Call > Accepted > Final; I don't see how non-core proposals should follow a different route.

File: README.md
Comment (soc1c) [reply README.md]:
```suggestion
* **Final** - An ECIP that was accepted, implemented, and no longer can be modified without submitting a new proposal, e.g., it has been released in a hard fork. 
```

File: README.md
Comment (ghost) [reply README.md]: Now that we simplified to just "Final" below, this line actually totally looks redundant so it deleted it.

File: README.md
Comment (ghost) [reply README.md]: Done.

Also I unified both "final" sections and used the text you suggested.

**Review by BelfordZ — APPROVED** [review]

**Review by soc1c — APPROVED** [review]

**Pull request merged**
